src/assets/translations/en/main.json

```diff
@@ -1445,6 +1445,22 @@
       "body": "By connecting a wallet, you agree to ShapeShift's Terms of Service and acknowledge that you have read and understand the ShapeShift disclaimer.",
       "footer": "Don't have a wallet?",
       "create": "Create one"
+    },
+    "walletConnectV2": {
+      "start": {
```

Comment:
Are we intentionally missing `body` here?

<img width="373" alt="Screenshot 2023-07-21 at 7 54 01 am" src="https://github.com/shapeshift/web/assets/97164662/7e471265-458d-4b2a-a4a8-b38b92bf9005">

Reply:
I didn't know what to put there, so left it blank. We can remove.